homeassistant/components/tuya/sensor.py

```diff
@@ -516,6 +516,31 @@ class TuyaSensorEntityDescription(SensorEntityDescription):
         ),
         *BATTERY_SENSORS,
     ),
+    # Smart Water Timer
+    "sfkzq": (
+        TuyaSensorEntityDescription(
+            key=DPCode.WORK_STATE,
+            name="Mode",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            translation_key="work_state",
+        ),
+        # Total seconds of irrigation. Read-write value; the device appears to ignore the write action (maybe firmware bug)
+        TuyaSensorEntityDescription(
+            key=DPCode.TIME_USE,
+            name="Total watering time",
+            icon="mdi:history",
+            state_class=SensorStateClass.TOTAL_INCREASING,
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
+        # Number of seconds for the watering session. The device will automatically turn off after this time
+        TuyaSensorEntityDescription(
+            key=DPCode.COUNTDOWN,
+            name="Countdown",
+            icon="mdi:timer-outline",
+            entity_category=EntityCategory.DIAGNOSTIC,
+        ),
```

Comment:
Does this count down / update the value during runtime? E.g., does this actually lower the value each update causing it to count down in the UI?

Reply:
Yes, it's updated

Reply:
That isn't allowed for a numeric sensor. It can provide a setting, but not an active countdown.

Reply:
Sorry @frenck but I don't understand the issue or in what way you want me to adjust this.
This is already being used in other device integrations like the [fan](https://github.com/home-assistant/core/blob/b2b57c5f87a519af951a9d1ca8777f5ae3517b92/homeassistant/components/tuya/select.py#L233C16-L233C16), [Air Purifier](https://github.com/home-assistant/core/blob/b2b57c5f87a519af951a9d1ca8777f5ae3517b92/homeassistant/components/tuya/select.py#L299). What is wrong with this one?

Reply:
It is allowed to update, but it isn't allowed to be used as a countdown.

This needs to be removed, otherwise it cannot be accepted.

Reply:
Ok I will remove this